uber/models.py

```diff
@@ -648,6 +648,9 @@ def change_badge(self, attendee, badge_type, badge_num=None):
         def valid_attendees(self):
             return self.query(Attendee).filter(Attendee.badge_status != c.INVALID_STATUS)
 
+        def all_attendees(self):
+            return self.staffers(only_staffing=False)
+
```

Comment:
So, agreed that ```self.staffers(only_staffing=False)``` is a really confusing name with a confusingly named argument.  But I do think it does indeed return ALL attendees.  The code is:

```            return (self.query(Attendee)
                        .filter(Attendee.badge_status.in_([c.NEW_STATUS, c.COMPLETED_STATUS]),
                                *[Attendee.staffing == True] if only_staffing else [])
                        .options(subqueryload(Attendee.group), subqueryload(Attendee.shifts).subqueryload(Shift.job))
                        .order_by(Attendee.full_name))```

which if only_staffing is false boils down to:

```            return (self.query(Attendee)
                        .filter(Attendee.badge_status.in_([c.NEW_STATUS, c.COMPLETED_STATUS]),
                        .options(subqueryload(Attendee.group), subqueryload(Attendee.shifts).subqueryload(Shift.job))
                        .order_by(Attendee.full_name))```

remember that this query is used by the email system to send out things like prereg confirmation emails, so it definitely has to already be working as-is for returning attendees who are not signed up for shifts.

My guess is the reason we're using this query is because it subloads Shifts, Groups, and Jobs, which some of the email categories use.

Can verify that this is true by running the following on production:
```
>>> session.query(Attendee).count()
10812
>>> session.staffers(only_staffing=False).count()
10812
```